src/cv_1.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="ru">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Конструктор резюме</title>
+    <link rel="stylesheet" href="./styles/reset.css">
+    <link rel="stylesheet" href="./styles/main.css">
+</head>
+<body>
+    <section class="blue">
+        <header class="blocking_heading">
+            <h1 class="h1 white">Конструктор резюме</h1>
+            <nav>
+                <a class="menu_btn input_head active" href="cv_1.html">Фото</a>
+                <a class="menu_btn input_head" href="cv_2.html">О себе</a>
+                <a class="menu_btn input_head" href="cv_3.html">Образование</a>
+                <a class="menu_btn input_head" href="cv_4.html">Опыт</a>
+                <a class="menu_btn input_head" href="cv_5.html">Дополнительно</a>
+            </nav>
+        </header>
+    </section>
+    <section class="blocking main">
+        <div class="input_line">
+            <h2 class="h2">Желаемая должность:</h2>
+            <input class="input" placeholder="PR-менеджер">
```

Comment:
У полей формы должен быть атрибут "name", чтобы [при отправке они попали в FormData](https://github.com/matiko-ma/cv_builder/pull/1#discussion_r2483889264):
```suggestion
            <input class="input" name="job_title" placeholder="PR-менеджер">
```